data/pizza.json

```diff
@@ -1248,7 +1236,7 @@
 			"ingredients": [
 				"kyckling", "champinjoner", "lök", "majs", "färska tomater", "gurka", "isbergssallad", "valfri sås"
```

Comment:
[Currysås instead of valfri](https://sannegarden.com/meny/johanneberg/#:~:text=Champinjoner%2C%20L%C3%B6k%2C%20Kyckling%2C%20Gurka%2C%20Isbergssallad%2C%20Majs%2C%20Currys%C3%A5s%2C%20Tomater)